LICENSE

```diff
@@ -200,3 +202,39 @@
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.
+
+------------------------------------------------------------------------------
+
+For Google protobuf source.
+
+file: src/packages/firestore/src/protos/google/protobuf/any.proto
```

Comment:
What should the path here?

Reply:
It's a relative path in google3, it's copied verbatim from the CL.

Reply:
But this license is for the public version of the SDK, should it use the path in Github?

Reply:
Unless we want to write a special script to reformat it on google3 import, it's probably best to make it applicable to wherever it is. I tried 
```
[source code root]/packages/firestore/src/protos/google/protobuf/any.proto
```